### PR TITLE
Forms for previous party

### PR DESCRIPTION
**Pull request by michaeljcollinsuk**
#1768

### CODE REVIEW

**Comment by michaeljcollinsuk** [comment]
I've left the commit history in place for now to show the thought process and changes in the approach  - however I think before merging I should squash down much of the changes

File: ynr/apps/candidates/static/js/person_form.js [hunk 1]
```diff
@@ -164,6 +165,12 @@ var populate_party_selects = function() {
   allPartySelects.attr("disabled", false);
   $(PARTY_WIDGET_INPUT_CLASS).hide();
   allPartySelects.each(setup_single_party_select);
+
```
Comment (VirginiaDooley) [inline ynr/apps/candidates/static/js/person_form.js:168]: @michaeljcollinsuk Just to confirm, this should be a multiple or single select element?

File: ynr/apps/candidates/static/js/person_form.js [hunk 2]
```diff
@@ -164,6 +165,12 @@ var populate_party_selects = function() {
...
+  $(PREVIOUS_PARTY_AFFILIATIONS_SELECT_CLASS).select2({
```
Comment (symroe) [inline ynr/apps/candidates/static/js/person_form.js:169]: I've not tested this locally, but does this allow multiple selections?

**Review by symroe — APPROVED** [review]
This is looking really good :+1:

File: ynr/apps/candidates/static/js/person_form.js (hunk 2 above)
Comment (michaeljcollinsuk) [reply ynr/apps/candidates/static/js/person_form.js:169]: Yes - can test on the bulk add form, it initialises all of the fields
Screenshot 2022-03-17 at 09 49 01

**Pull request merged**